pkg/iac/rego/load.go

```diff
@@ -127,6 +145,60 @@ func (s *Scanner) LoadPolicies(enableEmbeddedLibraries, enableEmbeddedPolicies b
 	return s.compilePolicies(srcFS, paths)
 }
 
+func (s *Scanner) fallbackChecks(compiler *ast.Compiler) {
+
+	var excludedFiles []string
+
+	for _, e := range compiler.Errors {
+		if _, ok := e.Details.(*ast.RefErrInvalidDetail); !ok {
+			continue
+		}
+
+		loc := e.Location.File
+
+		if lo.Contains(excludedFiles, loc) {
+			continue
+		}
+
+		badPolicy, exists := s.policies[loc]
+		if !exists || badPolicy == nil {
+			continue
+		}
+
+		if !IsBuiltinNamespace(getModuleNamespace(badPolicy)) {
+			continue
+		}
+
+		s.debug.Log("Error occurred while parsing: %s, %s. \nTry loading embedded policy.", loc, e.Error())
+
+		embedded := s.findMatchedEmbeddedCheck(badPolicy)
+		if embedded == nil {
+			s.debug.Log("Failed to find embedded policy: %s", loc)
+			continue
+		}
+
+		s.debug.Log("Found embedded policy: %s", embedded.Package.Location.File)
```

Comment:
We should use the term `check` going forwards.
```suggestion
		s.debug.Log("Found embedded check: %s", embedded.Package.Location.File)
```

Reply:
Done https://github.com/aquasecurity/trivy/pull/6502/commits/e247ad23a09fa850d6387a280b0fe1a2e153a83f